ui/legacyRouteHandler.js

```diff
@@ -59,5 +59,10 @@ export default function legacyRouteHandler(el) {
     return true;
   }
 
+  if ($('body').hasClass('service_types') && $('body').hasClass('is_service_working')) {
```

Comment:
For new things, it's better to put them into `App.js`, but no need to change this, we can just migrate this with the rest of them later as part of https://github.com/studentinsights/studentinsights/issues/1760.